kreyes-gourmet-cuisine/src/README.md

```diff
@@ -0,0 +1,81 @@
+#KReyes Gourmet Cuisine 
+
+##Description
+KReyes Gourmet Cuisine  is an application project for a restaurant, which consists of a user interface to order their food, the same client can build the desired food, as long as it is a variety plate, once the food is requested, an order is created that goes up to the kitchen, and the client will be able to follow up step by step to the elaboration of his food, observing until the remaining times.
+
+     
+### General flow
+                                           
+* The client accesses the application and looks for the food menu.
+* Choose the desired dish if it is available.
+* Choose the characteristics of your food such as: ingredients, accompaniments, etc.
+* Once you have chosen everything you want, the order is created.
+* The customer now has access to see the preparation process.
+* Once the order is ready, the customer is notified.
+* The customer can choose between picking up their food or receiving by delivery.
+
+##Setup
+
+This project is being developed in java and in order to start its development
+the following steps should be taken:
+
+1. It is suggested to create an [Oracle](https://profile.oracle.com/myprofile/account/create-account.jspx) account  as this facilitates many services
+and it allows us to download the desired version of the JDK, in case of not creating the account, you can search for a free version.
+
+2. Download and install the java developer kit [JDK_11](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html) in this case version 11 was chosen
+since it is very stable and maintained.
+
+3. Check that the environment variables are properly configured.
```

Comment:
What does this mean? what variables are you referring to? please specify this configuration